DigitalLearningSolutions.Web/Views/Home/LearningContent.cshtml

```diff
@@ -0,0 +1,75 @@
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+@{
+  ViewData["Title"] = "Welcome - Learning Content";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/learningContent.css")"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+
+    <vc:mini-hub-heading mini-hub-navigation-model="@Model.MiniHubNavigationModel" />

```

Comment:
Just a thought - are some of these VCs simple enough to be partial views instead? (an example can be found on LearningPortal/Current page)